src/main/webapp/WEB-INF/tags/ksr/securityRequestRole.tag

```diff
@@ -6,16 +6,21 @@
               description="Whether the Security Request Role data should be read-only" %>
 
 <c:set var="genericAttributes" value="${DataDictionary.AttributeReferenceDummy.attributes}" />
-
 <c:set var="securityRequestRole" value="${KualiForm.document.securityRequestRoles[securityRequestRoleIndex]}" />
+<c:set var="roleTitle" value="${securityRequestRole.roleInfo.id} : ${securityRequestRole.roleInfo.namespaceCode} - ${securityRequestRole.roleInfo.name}"/>
 
-<h3>${securityRequestRole.roleInfo.id} : ${securityRequestRole.roleInfo.namespaceCode} - ${securityRequestRole.roleInfo.name}</h3>
+<h3>&nbsp;</h3>
 <table width="100%" border="0" cellpadding="0" cellspacing="0" class="datatable">
+  <tr>
+    <th colspan="2" width="100%">
+      <b><c:out value="${roleTitle}"/></b>

```

Comment:
Is this title large enough, or does it need one of the numbered "h" tags to make it a little bigger? And is it being left-aligned or centered?

Reply:
the bold tag seemed to make it a visual focus to me.  I couldn't use the "h" tag because it would add a line below the text, which is Cathy didn't want

Reply:
Thanks for the explanation. This should be fine as-is.